### PR TITLE
Rspec tests

### PR DESCRIPTION
**Pull request by gabezurita**

### CODE REVIEW

File: fibonacci.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Fibonacci
+  def self.calculate(num)
+    if !(num.is_a? Integer)
+      return "😑 Fibonacci numbers must be integers--try again!"
+    elsif num < 0
+      return "🤷 The Fibonacci series starts at 0--try again!"
```
[the remark this answers is not in the document]
Comment (gabezurita) [reply fibonacci.rb]: Thanks for the feedback! I moved this to front-end form input validations and removed the model validations--I will add it back.

File: fibonacci.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      a = 0
+      b = 1
+      counter = 0
+
+      until counter == num
+        temp = a
+        a = b
+        b = temp + b
+        counter += 1
+      end
```
[the remark this answers is not in the document]
Comment (gabezurita) [reply fibonacci.rb]: I'll bring back the iterative solution in order to avoid recursive calls  :)

[the remark this answers is not in the document]
Comment (sfgeorge) [reply fibonacci.rb]: I know, it's hard to keep it iterative and keep it DRY at the same time, as you said!

Hope you had a great weekend.

[the remark this answers is not in the document]
Comment (gabezurita) [reply fibonacci.rb]: The weekend was great - I'm in Montreal on vacation! BTW, I made some updates based on the feedback you and Frederic gave, though I have not made it iterative again:
https://github.com/gabezurita/fibonacci_prime/pull/14

**Pull request merged**